src/models/course.ts

```diff
@@ -0,0 +1,30 @@
+import * as dynamoose from 'dynamoose';
+import { Document } from 'dynamoose/dist/Document';
+import CourseItem, { courseItemSchema, CourseItemInterface } from './courseItem';
+
+export const courseSchema = new dynamoose.Schema(
+    {
+        id: String,
+        name: String,
+        owner: String,
+        courseItems: {
+            type: Set,
+            schema: [CourseItem],
+        },
+    },
+    {
+        timestamps: true,
+    }
+);
+
+export interface CourseInterface extends Document {
```

Comment:
Should the course interface include a 'desiredGrade' field?

Reply:
That's a good question! I feel like it should, so I'll add that asap. :)